compiler/dyno/lib/parsing/ParserContextImpl.h

```diff
@@ -785,6 +786,173 @@ FunctionParts ParserContext::makeFunctionParts(bool isInline,
   return fp;
 }
 
+static AstNode*
+findErrorInFnBodyOrFormals(ParserContext* context, YYLTYPE location,
+                           FunctionParts& fp) {
+  if (fp.errorExpr) return fp.errorExpr;
+  if (fp.formals) {
+    for (auto ast : *fp.formals) {
```

Comment:
This function is called `findErrorInFnBodyOrFormals`, but I only see it checking the `fp.formals`. Does `fp.formals` contain the body somehow, or is the function name slightly inaccurate?

Reply:
Renamed it